runtime/gc_base/accessBarrier.cpp

```diff
@@ -835,11 +837,47 @@ j9gc_objaccess_jniDeleteGlobalReference(J9VMThread *vmThread, J9Object *referenc
 	barrier->jniDeleteGlobalReference(vmThread, reference);
 }
 
-void
+j9object_t
+synchronizeWithConcurrentGCScan(J9VMThread *currentThread, j9object_t continuationObject)
```

Comment:
I'd actually keep this at the old spot for now, so it's more clear what we changed. We can consider moving it to GC code within a separate change, possibly along with moving a couple of other methods/helpers.